policyengine_us/parameters/gov/hhs/tanf/cash/income/sources/earned.yaml

```diff
@@ -0,0 +1,13 @@
+description: The Department of Health and Human Services counts these income sources as earned income under the Temporary Assistance for Needy Families program.
+values:
+  2020-01-01:
+    - employment_income
+    - self_employment_income
```

Comment:
We might need to use *_before_lrs`, see [this](https://github.com/PolicyEngine/policyengine-us/pull/6740) PR

Reply:
Could you test the implementation, could you run a local  notebook with LSRs to see if that breaks